bcos-framework/bcos-framework/sync/SyncConfig.h

```diff
@@ -84,6 +84,18 @@ class SyncConfig
         return *m_observerNodeList;
     }
 
+    virtual void setWorkingConsensusList(bcos::consensus::ConsensusNodeList const& _workingNodeList)
+    {
+        WriteGuard lock(x_workingConsensusNodeList);
+        *m_workingConsensusNodeList = _workingNodeList;
+    }
+
+    virtual bcos::consensus::ConsensusNodeList workingConsensusList()
```

Comment:
```suggestion
    virtual bcos::consensus::ConsensusNodeList workingSealerNodeList()
```

why not use same name?